src/main/resources/templates/messages.ftlh

```diff
@@ -4,18 +4,34 @@
         <title>Messages</title>
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <script>
+            let getNotify = function() {
+                let xhr = new XMLHttpRequest();
+                xhr.responseType = "json";
+                xhr.open("GET", "/notify", true);
+                xhr.onreadystatechange = function() {
+                    if (xhr.readyState === 4 && xhr.status === 200) {
+                        console.log(xhr.response);
+                        if (xhr.response === true) {
```

Comment:
Isn't this possible?

```suggestion
                        if (xhr.response) {
```

Reply:
Me gustaría dejar así esa línea por:
1) Siento que es más legible. "_Si el `response` es `true`, entonces ejecuta_".
2) Puede que tu `response` cambie y te envié alguna otra cosa. Entonces quiero amarrar el hecho de que SOLO si trae `true` como valor, ejecute la siguiente línea.